src/Model/WhitelistedDomains.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Tgallice\FBMessenger\Model;
+
+use Tgallice\FBMessenger\Model\ThreadSetting\DomainWhitelisting;
+
+class WhitelistedDomains
```

Comment:
I'm not sure about the need of this class. Or It's should be more generic. Maybe rename `DomainWhitelisting` to `DomainWhitelist` and used it instead of `WhitelistedDomains`. The `DomainWhitelist` should contain only domain list.